src/components/icon/icon.component.js

```diff
@@ -79,11 +82,12 @@ const Icon = React.forwardRef(
       fontSize,
       isInteractive,
       iconColor,
-      tabIndex,
       type: iconType(),
       ...filterStyledSystemMarginProps(rest),
     };
 
+    const hasTooltip = tooltipMessage && !blockFocusBehaviour;
```

Comment:
Why does `blockFocusBehaviour ` change the `role`? 

Please can you update the PR description
> Sets ariaLabel to type if none passed and not composed as part of ValidationIcon or Help.

We should document the prop as well as the components it affects.

Reply:
Yeah good point I think it shouldn't affect role really, I had it this way as originally Help set the role, but then I spotted that this caused axe issues so moved the role to the Icon but missed reverting this